### PR TITLE
REL: bump to 2.1.10

### PR DESCRIPTION
**Pull request by wasade**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Bump for a minor bug fix impacting QIIME 2 dependency stack. cc @thermokarst

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "2.1.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 18a6e4d4b4b2a6bf2d5544fa357ad168bedeac93f0837015ef9c72f41fa89491
+  sha256: b63ce742dbe7d2353a48e004096645e9e2e4f7d53802e2c5d044f108b873a204
```
Comment (nsoranzo) [inline recipe/meta.yaml:10]: The sha256 here looks wrong. I've merged https://github.com/conda-forge/biom-format-feedstock/pull/16 from the bot which has the same string that I see on PyPI (and also does the re-render), so I will close this PR.

Comment (wasade) [reply recipe/meta.yaml:10]: Hi @nsoranzo, I computed it using the recommended [command](https://conda-forge.org/docs/maintainer/adding_pkgs.html#step-by-step-instructions)?

```bash
$ curl -sL https://github.com/biocore/biom-format/archive/2.1.10.tar.gz | openssl sha256
(stdin)= b63ce742dbe7d2353a48e004096645e9e2e4f7d53802e2c5d044f108b873a204
```

Comment (thermokarst) [reply recipe/meta.yaml:10]: @wasade, I think you're checking the github bundle in the snippet above, but the bundle that the recipe is using is coming from pypi:

```bash
$ curl -sL https://pypi.io/packages/source/b/biom-format/biom-format-2.1.10.tar.gz | openssl sha256
(stdin)= f5a277a8144f0b114606852c42f657b9cfde44b3cefa0b2638ab1c1d5e1d0488
```

Comment (wasade) [reply recipe/meta.yaml:10]: hah! ya, good catch :/

**Pull request closed without merging**